src/main.py

```diff
@@ -66,6 +69,10 @@ def setUp(self):
         """
         Start your UI and app run loop here.
         """
+        PythonActivity = autoclass("org.kivy.android.PythonActivity")
+        FullScreen = autoclass("org.learningequality.FullScreen")
+        configureWebview = Runnable(FullScreen.configureWebview)
+        configureWebview(PythonActivity.mActivity)
```

Comment:
This provides a great example for adding custom Java and tweaking the webview behavior without having to hack the P4A bootstrap. 🎆

Reply:
Yes, much thanks to @danigm for creating such a clean intervention! May help us to avoid having to fork P4A if we can follow this pattern more closely.